app/views/layouts/_header.html.slim

```diff
@@ -9,20 +9,20 @@ nav.navbar.navbar-inverse.navbar-fixed-top role="navigation"
       a.navbar-brand href=root_path  SpacialDB
     #navbar.collapse.navbar-collapse
       ul.nav.navbar-nav
-        li.active
+        li class=('active' if controller_name == 'home')
           a href="/"  Home
         li
           a href="#about"  About
         li
           a href="#contact"  Contact
-        li
+        li class=('active' if controller_name == 'instances')
           - if user_signed_in?
             a href="/instances"  Instances
```

Comment:
Truth... but just fix it while you are working on this file :wink: nothing wrong with improving it here